docs/en/serving-tiles/manually-building-a-tile-server-debian-13.md

```diff
@@ -0,0 +1,414 @@
+---
+layout: docs
+title: Manually building a tile server (Debian 13)
+dist: Debian 13
+dl_timestamp: "2025-08-15T12:40:00Z"
+lang: en
+---
+
+# {{ title }}
+
+!!! info ""
+    This page describes how to install, setup and configure all the necessary software to operate your own tile server. These step-by-step instructions were written for [Debian Linux](https://www.debian.org/){: target=_blank} 13 (trixie), and were tested in August 2025.
+
+## Software installation
+
+The OSM tile server stack is a collection of programs and libraries that work together to create a tile server. As so often with OpenStreetMap, there are many ways to achieve this goal, and nearly all of the components have alternatives that have various specific advantages and disadvantages. This tutorial describes the most standard version that is similar to that used on the main OpenStreetMap.org tile servers.
+
+It consists of 5 main components: `mod_tile`, `renderd`, `mapnik`, `osm2pgsql` and a `postgresql/postgis` database. Mod_tile is an apache module that serves cached tiles and decides which tiles need re-rendering&nbsp;– either because they are not yet cached or because they are outdated. Renderd provides a priority queueing system for different sorts of requests to manage and smooth out the load from rendering requests. Mapnik is the software library that does the actual rendering and is used by `renderd`.
+
+These instructions are have been written and tested against a {{ dist }} server on which all this software was newly installed. If you have got other versions of some software already installed (perhaps you upgraded from an earlier version some time ago, or you set up some PPAs to load from) then you may need to make some adjustments.
```

Comment:
I think this is still to complicated. How about
> These instructions are have been written and tested against a new {{ dist }} server.

Reply:
🤦 I must have been tired when I said "are have been written". That should be "are written"

Reply:
I've gone with "have been written".  Since last week I have now tested most of this on a fresh server.